docs/src/guide/installation.md

```diff
@@ -30,6 +30,40 @@ additional build system documentation.
 Please help us [keep these instructions up-to-date][contribute]!
 
 
+## Linux
+
+For Ubuntu 20.04LTS, these commands install the basic build system:
+
+	sudo apt-get install python3 python3-pip scons git gcc-arm-none-eabi openocd \
+	  gcc build-essential libboost-all-dev
+	pip3 install modm gdbgui
```

Comment:
What happened to avrdude etc?

Reply:
AVR Toolchain on Ubuntu is too old (GCC 5, modm requires >7 (?)).

You could leave the installation command for `avrdude`, but again I think someone will be able to do that if they can compile a modern AVR-GCC themselves...